mist/worker/src/main/scala/io/hydrosphere/mist/worker/Worker.scala

```diff
@@ -53,7 +56,7 @@ object WorkerArguments {
 }
 
 object Worker extends App with Logger {
-
+  val BaseDir = "/tmp"
```

Comment:
it's better to change it to `MIST_HOME`. Sometimes `tmp` may be mounted using `ramfs`